index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+
+<body>
+<header>Let's play rock, paper, scissors!</header>
+    <div id="select-rock">
+       <label>
+           <input name="rps" type="radio" value="rock">
+           <img src = "http://clipart-library.com/images_k/rock-clipart-transparent/rock-clipart-transparent-23.png">
+       </label> 
+    </div>
+    <div id="select-paper">
+        <label>
+            <input name="rps" type="radio" value="paper">
+            <img src = "https://i.dlpng.com/static/png/6347541_preview.png">
+        </label> 
+    </div>
+    <div id="select-scissors">
+        <label>
+            <input name="rps" type="radio" value="scissors">
+            <img src = "https://clipartix.com/wp-content/uploads/2016/04/Scissors-clip-art-cartoon-vector-by-igor-zakowski-image-1.jpg">
```

Comment:
if you are having issues with broken links  just save the asset on your computer, pull it into VScode, and link to it like you would another file.  There are a couple reasons why links can be broken but it's confusing to deal with right now!